policybot/dashboard/dashboard.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dashboard
+
+import (
+	"bytes"
+	"net/http"
+	"strings"
+	"text/template"
+
+	"github.com/gorilla/mux"
+
+	"istio.io/bots/policybot/dashboard/templates/layout"
+	"istio.io/bots/policybot/dashboard/templates/widgets"
+)
+
+type Dashboard struct {
```

Comment:
code comments on exported types and functions.

Reply:
DOne